clinica/iotools/bids_utils.py

```diff
@@ -435,19 +435,16 @@ def write_modality_agnostic_files(study_name, bids_dir):
     from os import path
 
     import clinica
+    from clinica.iotools.bids_dataset_description import BIDSDatasetDescription
 
-    dataset_dict = {"Name": study_name, "BIDSVersion": "1.4.1", "DatasetType": "raw"}
-    dataset_json = json.dumps(dataset_dict, skipkeys=True, indent=4)
-    f = open(path.join(bids_dir, "dataset_description.json"), "w")
-    f.write(dataset_json)
-    f.close()
+    with open(path.join(bids_dir, "dataset_description.json"), "w") as f:
+        BIDSDatasetDescription(name=study_name, bids_version="1.4.1").write(to=f)
```

Comment:
Didn't get why here there is `1.4.1` version while in the class definition (`clinica/iotools/bids_dataset_description.py `) there is a variable `BIDS_VERSION = "1.6.0"` ? (Some of our `dataset_description.json` files are `1.4.1` and others `1.6.0`. I'm probably missing something...).

Reply:
`BIDS_VERSION` should be updated to whatever the current BIDS standards version supported is (1.6.0 as of today). So a converter running today will output a `dataset_description.json` with a BIDS version set to 1.6.0. Recent converters we have worked on (NIFD, OASIS-3, HABS) are compliant with this version.

Now for the oldest ones (AIBL, ADNI, ...) had BIDS standards version set to `1.4.1`, which probably was the latest version _at the time they were implemented_. This PR does not assume their implementation is compliant with `1.6.0`, hence why the initial version of `1.4.1`is left. They are _probably_ compatible with `1.6.0`, but this verification is a topic for a different PR in my opinion.

Reply:
Most notably, version `1.6.0` introduced the definitive specification for PET imaging, whereas we used to do our own thing back then.

Reply:
Ok, I see. Thanks for the clarification!